.dotnet/Azure.AI.OpenAI.Staging/Custom/Audio/AzureAudioClient.Protocol.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using OpenAI;
+using OpenAI.Audio;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.ComponentModel;
+using System.Text;
+
+namespace Azure.AI.OpenAI.Staging.Audio;
+
+public partial class AzureAudioClient : AudioClient
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public override ClientResult TranscribeAudio(BinaryContent content, string contentType, RequestOptions options = null)
```

Comment:
Why do we have to override these methods as opposed to overriding just the CreateTranscriptionRequestMessage method?